src/coalescenceml/cli/stack.py

```diff
@@ -1,15 +1,25 @@
-from typing import Optional
```

Comment:
![image](https://user-images.githubusercontent.com/44331314/167268124-de6b1fe2-d45f-4fab-8132-176983b19b4e.png)
Container registry column name does not show